edge-util/src/Microsoft.Azure.Devices.Edge.Util/EventHubPartitionKeyResolver.cs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util
+{
+    using System;
+    using System.Globalization;
+    using System.Text;
+
+    public static class EventHubPartitionKeyResolver
```

Comment:
Can we move this to Test.Util?

Reply:
done (Moved under Edge.Test.Common as some of the referring places already referred that project and it also has a Partition related class already)